log4j-api/src/main/java/org/apache/logging/log4j/message/ParameterizedMessage.java

```diff
@@ -274,7 +278,12 @@ public void formatTo(final StringBuilder buffer) {
             buffer.append(formattedMessage);
         } else {
             final int argCount = args != null ? args.length : 0;
-            ParameterFormatter.formatMessage(buffer, pattern, args, argCount, patternAnalysis);
+            try {
+                ParameterFormatter.formatMessage(buffer, pattern, args, argCount, patternAnalysis);
+            } catch (final Exception error) {
+                STATUS_LOGGER.error("message formatting failure", error);
```

Comment:
Nit: the message is "Unable to format msg: {}" in the case of `StringFormattedMessage`. Can we use the same message everywhere?

Reply:
Fixed in a40714e415813433df099f6d15de5b511ea3f910.